### DISCUSSION TITLE
Add Relation list criteria

### DISCUSSION THREAD

**Discussion by paultvis** [atrocore/atrocore#543] [General]
Hi,

We are trying to move products into a category, but when we get the list of categories in Add Relation it only shows some of the categories we have. In total we have 722 categories across 3 catalogues. In the catalogue we are targeting we have 266, but the list only shows 232. Why does th list displayed not contain all categories? Is anyone able to explain what the selection criteria are for categories to be displayed in the add relation list?

Thanks

Paul

**Comment by atrolex** [comment]
Maybe you see only leaf categories and not all categories? It depends on configuration (/administation/pim), whether assignment to non-leaf categories is enabled or not.
